### DISCUSSION TITLE
Updating parent theme after project launch

### DISCUSSION THREAD

**Discussion by xeiter** [holger1411/understrap#2179] [Q&A]
Hi, I launched project a year ago using parent/child theme. Currently, my parent theme is Version: 1.1.0. 

It is offering me to update the parent theme to 1.2.2. What is the best practice about updating? Am I generally ok to update parent theme after project goes into production or do I leave it at the version that was current when I launched the project?

**Comment by bacoords** [comment]
Hey @xeiter Depending on how the child theme was built, you should usually be fine updating the parent theme. If you're unsure, I recommend making a staging copy of your site and testing the update there first.
